recipe/meta.yaml

```
@@ -45,7 +45,7 @@ test:
     - mocpy_test.py
   imports:
     - mocpy
-    - astropy
+    - astropy-base
```

This is a bug in the migration bot, bot I'm not sure it worth fixing centrally. cc @cadair @beckermr
```suggestion
    - astropy
```

---

(I mean, I don't think astropy should be even listed in here, so one could argue this is a recipe bug)

---

This line should not be in the recipe. Remove it and move on.